patches/server/0977-Add-PlayerGetRespawnLocationEvent.patch

```diff
@@ -0,0 +1,102 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Cryptite <cryptite@gmail.com>
+Date: Mon, 1 May 2023 15:23:34 -0500
+Subject: [PATCH] Add PlayerGetRespawnLocationEvent
+
+
+diff --git a/src/main/java/net/minecraft/server/players/PlayerList.java b/src/main/java/net/minecraft/server/players/PlayerList.java
+index 4d837c1530a3031a4c2a5a39d87bd013d60e14a6..2f07456c0ff0346b621d89bbf8d84d299daba00b 100644
+--- a/src/main/java/net/minecraft/server/players/PlayerList.java
++++ b/src/main/java/net/minecraft/server/players/PlayerList.java
+@@ -835,49 +835,57 @@ public abstract class PlayerList {
+ 
+         // CraftBukkit start - fire PlayerRespawnEvent
+         if (location == null) {
+-            // boolean isBedSpawn = false; // Paper - moved up
```

Comment:
This has a rather difficult to read diff, can you possibly try to reduce the diff/prevent changing indentation here?

Reply:
Fixed by not intending. Kind of atrocious but gets the job done?